shanoir-uploader/src/main/java/org/shanoir/uploader/service/rest/ShanoirUploaderServiceClient.java

```diff
@@ -521,6 +533,48 @@ public CloseableHttpResponse downloadDatasetsByStudyId(Long studyId, String form
 		return null;
 	}
 	
+	public Study createStudy(final Study study) {
```

Comment:
This method is never used, it is for a future usage to create a study automatically ?

Reply:
yes